deblur/workflow.py

```diff
@@ -427,6 +427,9 @@ def remove_artifacts_seqs(seqs_fp,
         else:
             sim_thresh = 0.65 * 100
 
+    # the minimal average bitscore per nucleotide
+    bitscore_thresh = 0.65
```

Comment:
There are a lot of parameters which are not exposed. In general, I agree with you @mortonjt and @ekopylova that it would be useful to facilitate exploration in the future but I think it may be easier to do this in a single shot as it would arguably make sense to package the user accessible parameters into an object. In other words, let's not do this now -- can you create an issue about this though?

Also, @mortonjt, if we change it in the future, it's either inline or the default value, so we don't get away from altering code somewhere.